h2o-algos/src/main/java/hex/tree/DHistogram.java

```diff
@@ -408,7 +421,7 @@ public void incr0( int b, double y, double w ) {
    * @param hi  upper bound on index into rows array to be processed by this call (exclusive)
    * @param lo  lower bound on index into rows array to be processed by this call (inclusive)
    */
-  public void updateHisto(double[] ws, double[] cs, double[] ys, int [] rows, int hi, int lo){
+  void updateHisto(double[] ws, double resp[], double[] cs, double[] ys, int [] rows, int hi, int lo){
```

Comment:
Please add doc. :) It is confusing to have `ys` named as response and add new value which is also named similar to response.

Reply:
Sure, will do - thank you.